docs/source/queries/queries.md

```diff
@@ -1,26 +1,80 @@
-# Making queries
-
-This driver supports all query types available in Scylla:
-* [Simple queries](simple.md)
-    * Easy to use
-    * Poor performance
-    * Primitive load balancing
-* [Prepared queries](prepared.md)
-    * Need to be prepared before use
-    * Fast
-    * Properly load balanced
-* [Batch statements](batch.md)
-    * Run multiple queries at once
-    * Can be prepared for better performance and load balancing
-* [Paged queries](paged.md)
-    * Allows to read result in multiple pages when it might be so big that one
-      prefers not to fetch it all at once
-    * Can be prepared for better performance and load balancing
-
-Additionally there is special functionality to enable `USE KEYSPACE` queries:
-[USE keyspace](usekeyspace.md)
-
-Queries are fully asynchronous - you can run as many of them in parallel as you wish.
+# Making queries - best practices
+
+Driver supports all kinds of statements supported by ScyllaDB. The following tables aim to bridge between DB concepts and driver's API.
+They include recommendations on which API to use in what cases.
+
+## Kinds of CQL statements (from the CQL protocol point of view):
+
+| Kind of CQL statement | Single              | Batch                                    |
+|-----------------------|---------------------|------------------------------------------|
+| Prepared              | `PreparedStatement` | `Batch` filled with `PreparedStatement`s |
+| Unprepared            | `Query`             | `Batch` filled with `Query`s             |
```

Comment:
Hmm, maybe it's worth mentioning that batch unprepared statements are being prepared by the driver if their bound value list is not empty. Actually, the same applies to single unprepared queries.